dashboards/gateway-api-inference-extension/inference-extension-prometheus.json

```diff
@@ -274,153 +341,197 @@
+                  "prometheusQuery": "histogram_quantile(0.90, sum by(le) (rate(inference_model_output_tokens_bucket{}[${__interval}])))"
                 }
               },
               {
-                "breakdowns": [],
-                "dimensions": [],
                 "legendTemplate": "p50",
-                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "outputFullDuration": false,
-                  "prometheusQuery": "histogram_quantile(0.5, sum by(le) (rate(inference_model_output_tokens_bucket{}[${__interval}])))",
-                  "unitOverride": ""
+                  "prometheusQuery": "histogram_quantile(0.5, sum by(le) (rate(inference_model_output_tokens_bucket{}[${__interval}])))"
+                }
+              },
+              {
+                "legendTemplate": "avg",
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "sum(rate(inference_model_output_tokens_bucket{}[${__interval}])) / sum(rate(inference_model_output_tokens_bucket{}[${__interval}]))\n"
+                }
+              },
+              {
+                "legendTemplate": "max",
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "histogram_quantile(1, sum by(le) (rate(inference_model_output_tokens_bucket{}[${__interval}])))"
+                }
+              },
+              {
+                "legendTemplate": "min",
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "histogram_quantile(0, sum by(le) (rate(inference_model_output_tokens_bucket{}[${__interval}])))"
                 }
               }
             ],
-            "thresholds": []
+            "thresholds": [],
+            "yAxis": {
+              "scale": "LINEAR"
+            }
           }
         }
       },
       {
-        "yPos": 73,
+        "yPos": 81,
         "height": 16,
         "width": 24,
         "widget": {
           "title": "Average KV Cache Utilization",
+          "id": "",
           "xyChart": {
             "chartOptions": {
               "displayHorizontal": false,
-              "mode": "COLOR"
+              "mode": "COLOR",
+              "showLegend": false
             },
             "dataSets": [
               {
+                "breakdowns": [],
+                "dimensions": [],
+                "legendTemplate": "",
+                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
+                  "outputFullDuration": false,
                   "prometheusQuery": "sum by (name)(avg_over_time(inference_pool_average_kv_cache_utilization[${__interval}]))*100",
```

Comment:
totally optional / just FYI: you can use a unit of `10^2.%` too, and then you don't need `*100` in queries.